### PR TITLE
Fixed multiline input style + validation style

### PR DESCRIPTION
**Pull request by gdostie**
screenshot from 2018-02-28 11-49-03

### CODE REVIEW

**Comment by sroy3** [comment]
Does the numeric spinner "With an error" and checkboxes "Validation on checkbox" still look fine?

File: scss/controls/input-field.scss [hunk 1]
```diff
@@ -172,6 +173,21 @@
     }
   }
 
+  // For multiline input validation feedback
+  .input-wrapper.validate > input {
+    & + label {
+      &:after {
+        margin-top: $input-message-position * 1.5;
```
Comment (pochretien) [inline scss/controls/input-field.scss]: why this 1.5 multiplicator ?

**Comment by gdostie** [comment]
@sroy3 est-ce que c'était comme ça avant ?

image

image

File: scss/controls/input-field.scss (hunk 1 above)
Comment (gdostie) [reply scss/controls/input-field.scss]: Without it:

multiline

With it:

multiline2

**Pull request merged**